src/main/scala/chisel3/util/Arbiter.scala

```diff
@@ -14,7 +14,10 @@ import chisel3.internal.naming.chiselName  // can't use chisel3_ version because
   * @param gen data type
   * @param n number of inputs
   */
-class ArbiterIO[T <: Data](gen: T, n: Int) extends Bundle {
+class ArbiterIO[T <: Data](private val gen: T, val n: Int) extends Bundle {
+  // gen is a val to allow autoclonetype to work, but private to not be detected as a Bundle field.
```

Comment:
Can you just link to the issue? (and just make this a one line comment) (1) It's easier to search for parts of the code related to the issue. (2) It's easier to get updates if someone sees this in the code.

Reply:
Good point, shortened the comment and linked the relevant issue.